superset-frontend/cypress-base/cypress/integration/chart_list/card_view.test.ts

```diff
@@ -109,7 +109,9 @@ describe('chart card view', () => {
     cy.get('[data-test="chart-list-edit-option"]').should('be.visible');
     cy.get('[data-test="chart-list-edit-option"]').click();
     cy.get('[data-test="properties-edit-modal"]').should('be.visible');
-    cy.get('[data-test="properties-name-input"]').should('not.have.value');
+    cy.get('[data-test="properties-nodal-name-input"]').should(
```

Comment:
What is `nodal`?

Reply:
A very embarrassing typo. That's what :P 

I _may_ have rushed that through just a tad.

Reply:
I mean... the whole PR is literally _one word_
![giphy (2)](https://user-images.githubusercontent.com/812905/97396422-49fdb900-18a4-11eb-8c4f-2826ca9b4b49.gif)